packages/server/tests/acceptance/rateLimiter.spec.ts

```diff
@@ -155,6 +158,46 @@ describe('@ratelimiter Rate Limiters Acceptance Tests', function () {
 
         await expect(relay.call(testConstants.ETH_ENDPOINTS.ETH_SEND_RAW_TRANSACTION, [signedTx], requestId)).to.be
           .fulfilled;
+        const remainingHbarsAfter = Number(await metrics.get(testConstants.METRICS.REMAINING_HBAR_LIMIT));
+        expect(remainingHbarsAfter).to.be.eq(remainingHbarsBefore);
```

Comment:
This seems off. 
I would expect the REMAINING_HBAR_LIMIT to have decreased for any call to `eth_sendRawTransaction`.
This is great that we're capturing it though

Reply:
This is correct. For transactions with small callData size the signer pays for the transaction. Only when for callData > 5120 do we use the `createFile + appendFile` flow. We cannot charge the signer for those transactions, because they are Hedera specific, and thus the relay operator pays for them.